ext/InverseFunctionsDistributionsExt.jl

```diff
@@ -0,0 +1,17 @@
+module InverseFunctionsDistributionsExt
+
+import InverseFunctions: inverse
```

Comment:
I'd prefer if functions are qualified explicitly when they are overloaded:
```suggestion
import InverseFunctions
```